include/pros/colors.h

```diff
@@ -168,4 +30,146 @@
 #define COLOR_LIGHT_SLATE_GREY COLOR_LIGHT_SLATE_GRAY
 #define COLOR_SLATE_GREY COLOR_SLATE_GRAY
 
-#endif  // _PROS_COLORS_H_
+typedef enum{
```

Comment:
nit: add spaces between "enum" or type name and the bracket wherever applicable.